frontend/src/__tests__/cypress/cypress/pages/modelServing.ts

```diff
@@ -226,6 +226,10 @@ class InferenceServiceModal extends Modal {
     return this.find().findByTestId('field AWS_S3_BUCKET');
   }
 
+  findBaseURL() {
```

Comment:
I have a PR adding some page objects that might be useful for you if you want to align with me https://github.com/opendatahub-io/odh-dashboard/pull/3867